packages/kbn-ftr-common-functional-services/services/saml_auth/serverless/auth_provider.ts

```diff
@@ -34,8 +34,18 @@ const getDefaultServerlessRole = (projectType: string) => {
   }
 };
 
+const isRoleManagementExplicitlyEnabled = (args: string[]): boolean => {
+  const roleManagementArg = args.find((arg) =>
+    arg.startsWith('--xpack.security.roleManagementEnabled=')
```

Comment:
It may simplify things to pull in the `getopts` package for parsing these.  I see `--serverless` is parsed similarly elsewhere, so don't see a need to change it now, but making a note in case we refactor later.